### ISSUE TITLE
Fix Travis Builds

### ISSUE THREAD

**Issue by EvanLovely** [phase2/drupal-lab#7]
Previous successful builds are now failing due to Composer 1.1.0 coming out and involves `composer-plugin-api`. Here's the error:

```
Loading composer repositories with package information
Installing dependencies (including require-dev) from lock file
Your requirements could not be resolved to an installable set of packages.
  Problem 1
    - Installation request for pattern-lab/unified-asset-installer v0.5.5 -> satisfiable by pattern-lab/unified-asset-installer[v0.5.5].
    - pattern-lab/unified-asset-installer v0.5.5 requires composer-plugin-api 1.0.0 -> no matching package found.
  Problem 2
    - pattern-lab/unified-asset-installer v0.5.5 requires composer-plugin-api 1.0.0 -> no matching package found.
    - pattern-lab/core v0.7.2 requires pattern-lab/unified-asset-installer ~0.5 -> satisfiable by pattern-lab/unified-asset-installer[v0.5.5].
    - Installation request for pattern-lab/core v0.7.2 -> satisfiable by pattern-lab/core[v0.7.2].
Potential causes:
 - A typo in the package name
 - The package is not available in a stable-enough version according to your minimum-stability setting
   see <https://getcomposer.org/doc/04-schema.md#minimum-stability> for more details.
Read <https://getcomposer.org/doc/articles/troubleshooting.md> for further common problems.
```

**Comment by EvanLovely** [comment]
OK, I've got a PR out that should fix this: https://github.com/pattern-lab/unified-asset-installer/pull/1

**Comment by EvanLovely** [comment]
PR #8 that I just merged fixes the by locking composer down to v1.0.3 in our build script used by Travis and devs.
